website/react-magma-docs/src/pages/api/popover.mdx

```diff
@@ -112,29 +112,63 @@ import {
   PopoverContent,
   PopoverApi,
   Button,
+  Spacer,
 } from 'react-magma-dom';
 
 export function Example() {
-  const popoverApiRef = React.useRef<PopoverApi>();
+  const closePopoverApiRef = React.useRef<PopoverApi>();
+  const openPopoverApiRef = React.useRef<PopoverApi>();
 
   function handleClose(event: React.SyntheticEvent) {
-    if (popoverApiRef.current) {
-      popoverApiRef.current.closePopoverManually(event);
+    if (closePopoverApiRef.current) {
+      closePopoverApiRef.current.closePopoverManually(event);
     }
   }
 
+  function handleOpen(event: React.KeyboardEvent) {
+    openPopoverApiRef.current.openPopoverManually(event);
+  }
+
   return (
-    <Popover apiRef={popoverApiRef}>
-      <PopoverTrigger aria-label="Toggle popover">
-        Toggle Popover
-      </PopoverTrigger>
-      <PopoverContent>
-        <div>
-          Popover Content
-          <Button onClick={handleClose}>Close Popover</Button>
-        </div>
-      </PopoverContent>
-    </Popover>
+    <div style={{ display: 'flex', flexDirection: 'column', gap: 8 }}>
+      <Popover apiRef={closePopoverApiRef}>
+        <PopoverTrigger aria-label="Toggle popover">
+          Toggle Popover
+        </PopoverTrigger>
+        <PopoverContent>
+          <div>
+            Popover Content
+            <Button onClick={handleClose}>Close Popover</Button>
+          </div>
+        </PopoverContent>
+      </Popover>
+      <Spacer axis={SpacerAxis.vertical} size={16} />
```

Comment:
I think we can remove this. The example below covers both `openPopoverManually` and `closePopoverManually`